modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsCalloutFieldMerge.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Drupal\az_migration\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Process plugin to merge field for callout paragraphs.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "paragraphs_callout_field_merge"
+ * )
+ */
+class ParagraphsCalloutFieldMerge extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    // Merging the data into one field.
+    // Backgroud Class.
+    $bg_light = '';
+    if (!empty($this->configuration['background'])) {
+      $field_uaqs_callout_background = $row->getSourceProperty($this->configuration['background']);
+      if (!empty($field_uaqs_callout_background[0]['value'])) {
+        $bg_light = 'bg-light';
+      }
+    }
+    // Color and text Mapping.
+    $field_uaqs_callout_border_color = $row->getSourceProperty('field_uaqs_callout_border_color');
```

Comment:
I think the `border_color` field should be passed via configuration, similar to the other fields that are being used.

Reply:
yes, done.